other/field-types/joinByArray.md

```diff
@@ -0,0 +1,115 @@
+
+When you specify the `relationship` property, you **may** also specify `relationshipsField`, a property name to store the relationships in. If you do not specify this property, it is set automatically. For instance, if the join is named `_departments`, the relationships will be stored in `departmentsRelationships`.
+
+### Example 2
+
+```javascript
+{
+  name: '_departments',
+  label: 'Departments',
+  // We could skip this since it is the same as the name of the join,
+  // plus an s. Since we specified it, it must match the `name` property
+  // of another type (it'll be singular)
+  withType: 'department',
+  type: 'joinByArray',
+  filters: {
+    // Fetch just enough information
+    projection: {
+      title: 1,
+      _url: 1
+    }
+  },
+  relationship: [
+    {
+      name: 'jobTitle',
+      label: 'Job Title',
+      type: 'string'
+    }
+  ]
+}
+```
+
+Since there is a relationship, the `_departments` property will be an array of objects with `item` and `relationship` properties. The `item` property will be the actual department, and the `relationship` property will contain the relationship fields, which are unique to this pairing.
+
+###Inline relationship fields
+
+Sometimes, expecting users to click a special button to access a separate modal dialog box to edit relationship fields isn't worth it. Users just don't find it, or the fields are few enough that it would make more sense to add the form field directly to the chooser.
+
+You can do this with the `inline: true` flag \(since 2.6.0\):
+
+### Example 3
+
+```javascript
+relationship: [
+  {
+    name: 'jobTitle',
+    label: 'Job Title',
+    type: 'string',
+    inline: true
+  }
+],
+```
+
+If you have a mix of inline and regular fields, you'll still get the option of opening the modal, but for data integrity reasons fields are presented only in one place or the other.
+
+## Settings
+
+|  Property | Type   | Default | Description | Sub-property |
+|---|---|---|---|
+|name | `string` | | Sets the name of the field in the database | |
+|label | `string` | | Sets the label of the field that the user sees | |
+|required | `boolean` | false | If true, the field is mandatory | |
+|type | `string` | | Specifies the field type | |
+|readOnly | `boolean` | false | If true, prevents the user from editing the field | |
+|help | `string` | | Help text for the field that will appear with the field's label | |
+|htmlHelp | `string` | | Help text with support for HTML markup | |
+|withType | `string` | | The name of the related type, if it differs from the name of the join. If you do not set `withType`, then the name of the join must match the name of the related type, with a leading `_` added.  | |
+|idsField | `string` | | Sets the name of the property in which to store the id. The id is set automatically otherwise. | |
```

Comment:
The name of the property is chosen automatically otherwise.

Reply:
Also, ids not id.

Reply:
Updated description and "id" -> "ids"